static/js/containers/App.js

```diff
@@ -88,49 +85,19 @@ class App extends React.Component {
       location: { pathname },
       dispatch,
     } = this.props;
-    let [ complete, info ] = validateProfileComplete(profile);
+    let [ complete, url, errors] = validateProfileComplete(profile);
     if (
       userProfile.getStatus === FETCH_SUCCESS &&
       profile.agreed_to_terms_of_service &&
       !PROFILE_REGEX.test(pathname) &&
       !complete
     ) {
-      const { url, title, text } = info;
-      dispatch(updateDialogText(text));
-      dispatch(updateDialogTitle(title));
-      dispatch(setDialogVisibility(true));
+      dispatch(startProfileEdit);
```

Comment:
@aliceriot error object looks good. I checked it. 
there is on error on console that appears on pressing ``save and continue`` on personal tab

**undefined:1 Uncaught (in promise) undefined**